Resources/views/CRUD/base_edit_form.html.twig

```diff
@@ -35,7 +35,12 @@
                         <div class="nav-tabs-custom">
                             <ul class="nav nav-tabs" role="tablist">
                                 {% for name, form_tab in admin.formtabs %}
-                                    <li{% if loop.index == 1 %} class="active"{% endif %}><a href="#tab_{{ admin.uniqid }}_{{ loop.index }}" data-toggle="tab"><i class="fa fa-exclamation-circle has-errors hide"></i> {{ admin.trans(name, {}, form_tab.translation_domain) }}</a></li>
+                                    <li{% if loop.first %} class="active"{% endif %}>
+                                        <a href="#tab_{{ admin.uniqid }}_{{ loop.index }}" data-toggle="tab">
+                                            <i class="fa fa-exclamation-circle has-errors hide"></i>
+                                            {{ admin.trans(form_tab.label, {}, form_tab.translation_domain) }}
```

Comment:
For those wondering, the thing that changed here is `form_tab.label` (is there something else @EmmanuelVella ?). Please next time do separate commits with `git add --patch`, otherwise it's quite hard to read.